packages/components/src/core/Tag/index.stories.tsx

```diff
@@ -102,102 +102,241 @@ export const Default = {
+  columnGap: "20px",
+  marginRight: "50px",
 };
+const displayContents = {
+  display: "contents",
+};
+const typeLevel = {
+  display: "grid",
+  gridTemplateColumns: "min-content min-content",
+  columnGap: "10px",
+};
+const iconLevel1 = {
+  gridArea: "1 / 2 / 2 / 3",
+};
+const iconLevel2 = {
+  gridArea: "2 / 3 / 2 / 3",
+};
+
+const styleLabel = {
+  fontWeight: "normal",
+  gridColumn: "2 / 4",
+  margin: 0,
+};
+const colorLabel = {
+  fontWeight: "normal",
+  gridColumn: "2 / 4",
+  hyphens: "auto",
+  borderStyle: "solid none none none",
+  justifySelf: "stretch",
+  margin: "40px 0 0 0",
+  paddingTop: "10px",
+};
+const stateLabel = {
+  fontWeight: "normal",
+  gridColumn: "2 / 4",
+  borderStyle: "solid none none none",
+  borderWidth: "1px",
+  justifySelf: "stretch",
+  margin: "20px 0 10px 0",
+  paddingTop: "10px",
+};
+const hoverLabel = {
+  fontWeight: "normal",
+  gridColumn: "1 / 2",
+  alignSelf: "end",
+  margin: "0 0 20px 0",
+};
+const typeLabel = {
+  fontWeight: "normal",
+  gridColumn: "1 / 3",
+  margin: "10px 0",
+};
+const iconLabel = {
+  fontWeight: "normal",
+  margin: "0 0 10px 0",
+};
+
+const SDS_STYLES = ["rounded", "square"];
+const COLORS = [
+  "primary",
+  "info",
+  "success",
+  "warning",
+  "error",
+  "gray",
+  "beta",
+  ["#A52A2A", "#ffa07a", "#ffe74a"],
+];
+const PSEUDO_STATES = ["default", "hover", "active", "focus-visible"];
+const HOVERABLE_OPTIONS = [true, false];
+const SDS_TYPES = ["primary", "secondary"];
+const ICON_OPTIONS = [
+  undefined,
+  <Icon sdsSize="l" sdsIcon="checkCircle" sdsType="button" />,
+];
 
 export const LivePreview = {
   args: {
+    sdsStyle: SDS_STYLES[0],
+    color: COLORS[0],
+    hover: HOVERABLE_OPTIONS[0],
+    sdsType: SDS_TYPES[0],
     label: "Label",
   },
   parameters: {
     snapshot: {
       skip: true,
     },
   },
-  render: (args: Args) => <LivePreviewDemo {...args} />,
+
+  render: (props: Args): JSX.Element => {
+    // loop through all SDS_STYLES
+    return (
+      <>
+        {SDS_STYLES.map((sdsStyle) => {
+          return <TagStyle sdsStyle={sdsStyle} key={sdsStyle} />;
+        })}
+      </>
+    );
+
+    // loop through all COLORS + create headers for SDS_STYLES
+    function TagStyle({ sdsStyle }) {
+      return (
+        <div style={styleLevel}>
+          <h1 style={styleLabel}>
+            Style: <b>{sdsStyle}</b>
+          </h1>
+          {COLORS.map((color) => {
+            return (
+              <TagHoverableOption
+                sdsStyle={sdsStyle}
+                color={color}
+                key={color}
+              />
+            );
+          })}
+        </div>
+      );
+    }
+
+    // loop through all PSEUDO_STATES + create headers for COLORS
+    function TagHoverableOption({ sdsStyle, color }) {
+      return (
+        <div style={displayContents}>
+          <h2 style={colorLabel}>
+            Color: <b>{color[0].length === 1 ? color : "custom"}</b>
```

Comment:
This creates the text heading for each color. There was an issue with the spacing of the grid when I used the `{color}` var exclusively as the content, because the custom color, which is defined in the `COLOR` const as a sub-array of actual hex values (but the other colors are named vars) "prints out" to the UI as the three hex codes all in a row with no spaces (#A52A2A#ffa07a#ffe74a). It will not break when I tried to the CSS `hyphen: auto` so it was making a very long non-breaking word which resulted in spacing out the distance between the tags more than intended. Also, it is more useful to label this color as "Custom" rather than the hexcodes, so in this line I tried to query whether the `color` var per loop was an array but it was not working... the only thing I could figure out was to get the length of the `color` item and if it is just a length of one then it's just one word (so a predefined SDS color) rather than the custom sub-array. So point being, this line is checking whether the `color` is the custom color or not, and if so it labels it as "Custom" in the UI.